lms/djangoapps/certificates/signals.py

```diff
@@ -159,18 +156,3 @@ def _listen_for_enrollment_mode_change(sender, user, course_key, mode, **kwargs)
                 course_key,
             )
             return False
-
-
-@receiver(CERTIFICATE_CREATED)
-def listen_for_certificate_created_event(sender, signal, **kwargs):
-    """
-    Publish `CERTIFICATE_CREATED` events to the event bus.
-    """
-    if SEND_CERTIFICATE_CREATED_SIGNAL.is_enabled():
```

Comment:
Maybe you could share the private PR for edx-internal settings so we can cross-reference that the old settings are being replaced by the new, and that they match values (enabled vs disabled)?

Reply:
Share it where? I don't think it makes sense to link it on a public PR.

Reply:
If you don't want to add private links here, maybe we just add more details to the deployment plan document?